### DISCUSSION TITLE
NRCAN Seedlink not working

### DISCUSSION THREAD

**Discussion by CSEtm** [xspanger3770/GlobalQuake#303] [Announcements]
The seedlink for NRCAN earthquakescanada.nrcan.gc.ca 18000, and UFRN (Universidade Federal do Rio Grande do Norte) sislink.geofisica.ufrn.br 18000. Is there new links for the stations?  Thank you for your time.

**Comment by YacineBoussoufa** [comment]
Canada is just bein Canada, it randomly dies and returns. Just update it 2/3 times and will appear back. For UFRN they closed they Seedlink and we don't have a new one. 
Keep in mind that GlobalQuake doesn't own any form of data, and the respective owners can stop sharing them via Seedlink Server and/or FDSNWS at any moment without notice.

**Comment by CSEtm** [comment]
Great thank you 😀

On Thu, Feb 29, 2024 at 9:04 AM Yacine Boussoufa ***@***.***>
wrote:

> Canada is just bein Canada, it randomly dies and returns. Just update it
> 2/3 times and will appear back. For UFRN they closed they Seedlink and we
> don't have a new one.
> Keep in mind that GlobalQuake doesn't own any form of data, and the
> respective owners can stop sharing them via Seedlink Server and/or FDSNWS
> at any moment without notice.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/xspanger3770/GlobalQuake/issues/256#issuecomment-1971457734>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AD2ZXS42G6PM2FXR2DCTDODYV5IPZAVCNFSM6AAAAABEAGTJRWVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMYTSNZRGQ2TONZTGQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*NANDINI LLC*
*20404 S W 111th St*
*Oklahoma City, OK 73170-5805*
*Corporate Manager*

*Curtis Ewers**Cell Phone: (719) 429-2615*

Days Inn By Wyndham of Custer
(605) 782-9130
Address: 519 Crook St, Custer, SD 57730
*---------------------------------------------------------------*
Soda Butte Lodge
(406) 838-2251
Address: 210 E Main St, Cooke City, MT 59020

This email message (including all attachments) is for the sole use of the
intended recipient(s) and may contain confidential information. If you are
not the intended recipient, please contact the sender by reply email and
destroy all copies of the original message. Unless otherwise indicated in
the body of this email, nothing in this communication is intended to
operate as an electronic signature and this transmission cannot be used to
form, document, or authenticate a contract. Wyndham Hotels and Resorts
and/or its affiliates may monitor all incoming and outgoing email
communications in the United States, including the content of emails and
attachments, for security, legal compliance, training, quality assurance
and other purposes.

**Comment by CSEtm** [comment]
I'm still not getting the earthquake information for California area?  I see stations, and they are blue and green, but I keep getting notifications for California thru USGS but not on your app?
